app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -0,0 +1,98 @@
+<ng-form name="persistentVolumeClaimForm">
+  <fieldset ng-disabled="routingDisabled">
+
+    <!-- Name -->
+    <div ng-show="showNameInput" class="form-group">
+      <label for="claim-name" class="required">Name</label>
+      <!-- regex, maxlength from k8s.io/kubernetes/pkg/util/validation/validation.go -->
```

Comment:
I don't think the path to validation.go is correct anymore. Maybe just remove the comment here and below.